protocol/x/clob/keeper/clob_pair.go

```diff
@@ -165,19 +168,6 @@ func (k Keeper) createOrderbook(ctx sdk.Context, clobPair types.ClobPair) {
 // CreateClobPair creates a new `ClobPair` in the store and creates the corresponding orderbook in the memclob.
 // This function returns an error if a value for the ClobPair's id already exists in state.
 func (k Keeper) CreateClobPair(ctx sdk.Context, clobPair types.ClobPair) error {
-	// Validate the given clob pair id is not already in use.
-	if _, exists := k.GetClobPair(ctx, clobPair.GetClobPairId()); exists {
-		panic(
-			fmt.Sprintf(
-				"ClobPair with id %+v already exists in state",
-				clobPair.GetClobPairId(),
-			),
-		)
-	}
-
-	// Write the `ClobPair` to state.
-	k.setClobPair(ctx, clobPair)
```

Comment:
nit: Why can't this call still be in here?

Reply:
Just refactoring so that the chain state setting functions are in a diff place than the funcs that set in memory state

Reply:
Then comment / naming needs to be updated otherwise it will be confusing.